llvm/docs/HowToReleaseLLVM.rst

```diff
@@ -328,17 +328,17 @@ Release Patch Rules
 Below are the rules regarding patching the release branch:
 
 #. Patches applied to the release branch may only be applied by the release
-   manager, the official release testers or the code owners with approval from
+   manager, the official release testers or the maintainers with approval from
```

Comment:
Unrelated side note: I think that nowadays only release managers can merge to release branches.